terraform/aws/modules/kubernetes/deployments/ingress.tf

```diff
@@ -12,12 +12,12 @@ resource "kubernetes_ingress_v1" "three_edges_ingress" {
     ingress_class_name = "nginx"
 
     tls {
-      hosts       = ["*.${var.hosted_zone}"]
+      hosts       = [var.hosted_zone, "*.${var.hosted_zone}"]
       secret_name = "letsencrypt-wildcard-secret"
     }
 
     rule {
-      host = var.hosted_zone
+      host = "frontend.${var.hosted_zone}"
```

Comment:
This "frontend" was a test, you can remove it from: Pod, Service and Ingress

Reply:
I have removed the frontend pod, svc, ingress rule